blocks/init/src/Blocks/custom/lottie/lottie.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/**
+ * Template for the Lottie Component.
+ *
+ * @package %g_namespace%
+ */
+
+use %g_namespace%\EightshiftLibs\Helpers\Helpers;
+
+$manifest = Helpers::getManifestByDir(__DIR__);
+
+$additionalClass = $attributes['additionalClass'] ?? '';
+$blockClass = $attributes['blockClass'] ?? '';
+$blockJsClass = $attributes['blockJsClass'] ?? '';
+$blockClass = $manifest['blockClass'] ?? '';
+$selectorClass = $attributes['selectorClass'] ?? $blockClass;
+$blockJsClass = $manifest['blockJsClass'] ?? '';
+
+$lottieUse = Helpers::checkAttr('lottieUse', $attributes, $manifest);
+$lottieTriggerResponsive = Helpers::checkAttrResponsive('lottieTrigger', $attributes, $manifest);
```

Comment:
I like to keep variable names the same as attribute names when ever that is possible so please check if this is possible